### DISCUSSION TITLE
Apple Game Mode

### DISCUSSION THREAD

**Discussion by QuasarRyan** [PrismLauncher/PrismLauncher#3127] [Ideas]
There's a new feature in macOS named Game Mode and I wonder if there's any possible to add support for it. The Minecraft Launcher has already support it.

**Comment by kthchew** [comment]
#1644 (tl;dr can't be implemented by Prism devs without changes to macOS by Apple)

IMO easiest user-level workaround currently is https://github.com/PrismLauncher/PrismLauncher/issues/1644#issuecomment-2508796311 (YMMV if you change the code signature like in some of the other solutions, and it might cause issues with resource access permissions and whatnot)
